analyticsdataserver/settings/local.py

```diff
@@ -62,6 +62,8 @@
     'api_key': 'edx'
 }
 
+ENABLE_ENTERPRISE_INTEGRATION = True
```

Comment:
I question the value of this setting. I could see an argument being made for having a waffle switch that would allow you to turn the enterprise-related endpoints on/off at runtime. Doing it at deployment doesn't seem all that useful.